Godeps/Godeps.json

```diff
@@ -117,28 +123,65 @@
 		{
 			"ImportPath": "github.com/golang/glog",
 			"Rev": "d1c4472bf2efd3826f2b5bdcc02d8416798d678c"
+			"ImportPath": "github.com/gogo/protobuf/proto",
+			"Rev": "5ba1012ef3868db1b9ff374280a558c1bfff39c3"
 		},
 		{
 			"ImportPath": "github.com/golang/mock/gomock",
 			"Rev": "15f8b22550555c0d3edf5afa97d74001bda2208b"
+			"ImportPath": "github.com/golang/glog",
+			"Rev": "d1c4472bf2efd3826f2b5bdcc02d8416798d678c"
 		},
 		{
 			"ImportPath": "github.com/golang/protobuf/proto",
 			"Rev": "c22ae3cf020a21ebb7ae566dccbe90fc8ea4f9ea"
 		},
 		{
+			"ImportPath": "github.com/hashicorp/go-msgpack/codec",
+			"Rev": "fa3f63826f7c23912c15263591e65d54d080b458"
+		},
+		{
+			"ImportPath": "github.com/hashicorp/raft",
+			"Rev": "53ca2ec750f09e888a0c051e7c68c98246176be2"
+		},
+		{
+			"ImportPath": "github.com/hashicorp/raft-boltdb",
+			"Rev": "d1e82c1ec3f15ee991f7cc7ffd5b67ff6f5bbaee"
+		},
+		{
 			"ImportPath": "github.com/influxdb/influxdb/client",
-			"Comment": "v0.8.0-rc.3-9-g3284662",
-			"Rev": "3284662b350688b651359f9124928856071bd3f5"
+			"Comment": "v0.9.1",
+			"Rev": "8b3219e74fcc3843a6f4901bdf00e905642b6bd6"
 		},
 		{
 			"ImportPath": "github.com/kr/pretty",
 			"Comment": "go.weekly.2011-12-22-20-g088c856",
 			"Rev": "088c856450c08c03eb32f7a6c221e6eefaa10e6f"
+			"ImportPath": "github.com/influxdb/influxdb/influxql",
+			"Comment": "v0.9.1",
+			"Rev": "8b3219e74fcc3843a6f4901bdf00e905642b6bd6"
```

Comment:
These imports are not valid json. Different imports need to be in separate objects.